.appveyor.yml

```diff
@@ -48,7 +48,7 @@ install:
   - cinst StrawberryPerl --version %perl% %x86% --allow-empty-checksums
   - refreshenv
   - perl -V
-  - perl -e "print qq(o conf urllist push http://www.cpan.org\no conf commit\n)" | cpan
+  - perl -e "print qq(o conf urllist push http://www.cpan.org\n)" | cpan
```

Comment:
I tried it on my machine and it doesn't seem to commit. Are you sure it commits always?

Reply:
It looks like that on AppVeyor before this change is _commit_ called two times:
https://ci.appveyor.com/project/perl5-dbi/dbd-mariadb/builds/47523009/job/watsk1heaywf61ep#L88

And with this change is _commit_ called just once:
https://ci.appveyor.com/project/perl5-dbi/dbd-mariadb/builds/47523083/job/mjliptfb27d0lur3#L89

So I was in impression that `urllist push` must do implicit `commit`. Hm... maybe implicit commit is done after exiting from cpan command? Or AppVeyor cpan is different?

Reply:
Maybe it has
```
o conf auto_commit 1
```

Reply:
Good catch, it looks like that StrawberryPerl has enabled cpan auto_commit by default:
https://github.com/StrawberryPerl/Perl-Dist-Strawberry/blob/master/share/config-files/CPAN_Config.pm.tt#L27

Reply:
I updated commit message.